src/main/java/com/wiley/autotest/selenium/context/OurSearchStrategy.java

```diff
@@ -41,7 +57,7 @@ public OurSearchStrategy nullOnFailure() {
     }
 
     public long getTimeout() {
```

Comment:
@estuold  Actually I did it as "getCustomTimeout" at firsr... but then here's why I changed it
OurSearchStrategy as an object has only one timeout (even though i has 2 fields 'timeout' and 'customTimeout' . - kind of a bad design but works for now) 
so I decided to keep the getter simple so users don't have to think "custom" or "not-custom".  Does it make sense? If no - let's discuss in gitter ;) @atyukavkin what do you think?